packages/taquito-rpc/src/rpc-client-modules/rpc-cache.ts

```diff
@@ -577,11 +528,8 @@ export class RpcClientCache implements RpcClientInterface {
   }
 
   /**
-   *
-   * @param options contains generic configuration for rpc calls
-   *
+   * @param options contains generic configuration for rpc calls to specified block (default as head) and version
```

Comment:
some grammar nitpicks, 
- to a specified block
- default is head/defaults to head

same comment to other instances of this

Reply:
appreciate this, it's addressed in new commit